.github/workflows/linting.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, and run linting
+
+name: Linting CI
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+
+jobs:
+  build:
+    name: Run linters and check build scripts
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
```

Comment:
Eventually we should probably update this project to Node 18 which is the current LTS. Although no rush since Node 16 is still in maintenance support until fall 2023

Reply:
I found that the current version of storybook doesn't support above node 16. I know there are newer versions so that's probably worth looking into as well.